docs/dm/assets.md

```diff
@@ -0,0 +1,170 @@
+If already in the initiative menu, this will appear as “show initiative”.
+For more information, check the [initiative](/docs/tools/intiative.md) document.
+
+### Delete shapes
+
+This option deletes your asset.
+
+### Show properties
+
+This option opens the “Edit asset” menu, described below.
+
+### Set marker
+
+This option sets the asset as a marker.
+When an object is added as a marker, it can be accessed from the dropdown menu.
+To access the Markers menu, click <font-awesome :icon="['far', 'compass']"/> and “Markers”.
+In the Markers menu, you can click the object to quickly snap your view to that object.
+
+## Edit asset
+
+![](/docs/dm/edit-asset.png)
+
+By right clicking and selecting the “Show properties” option, you open the “Edit asset” menu.
+This menu includes a plethora of options.
+
+### Is a token
+
+Selecting this option classifies the asset as a token. 
+This will help you keep track of which assets are tokens and which are other assets.
+
+### Is invisible
+
+Selecting this option turns the asset invisible to your players.
+As the DM, you will still be able to see the asset.
+This can be useful for hiding future encounters or placing enemies in stealth. 
+
+### Is locked
+
+Selecting this option locks the object in place.
+This can be useful for maps you don’t want to accidently move while playing.
+
+### Show badge
+
+Selecting this option displays a badge at the bottom right of the object.
+
+### Blocks vision/light
+
+Selecting this option blocks vision and light from passing through the object.
+More information about lighting can be found in the [lighting and vision](/docs/dm/light-shadows.md) document.
+
+### Blocks movement
+
+Selecting this option prevents other objects from moving through it.
+This is useful for creating walls in your dungeon you don’t want your players to pass through.
+
+### Border colour & Fill colour
+
+These options control the color of the borders around and the fill color of default tokens. 
+
+### Access
+
+Here is listed the players who have access to the asset.
+Selecting a player name and “add access” will allow them to interact with the asset.
+Selecting any of the icons next to “default” gives all players access to the asset.
+The pen icon allows those with access to edit the asset.
+The four arrows icon allows those with access to move the asset unless locked.
+The lightbulb icon allows those with access to see the asset. 
+
+### Trackers
+
+Trackers can track numerical values of assets, such as health to player tokens.
+To use, under “name” label the value you wish to track, such as health.
+Next, write the current value which the player has and the maximum value.
+By clicking the eye icon, you can toggle whether this information is public to all players or strictly visible to the DM.
+
+With these values established, when you click the asset, you will now see the values in a quick menu.
+You can quickly edit these values by clicking the number and typing in the new value.
```

Comment:
As hinted at by dthv, the update field also accepts values like `-10` or `+5` to do quick simple subtraction/addition

Reply:
I actually never managed to get this to work. How should it work? Just clicking into the respective field and typing `+5`? I should probably file a bug, then.